### DISCUSSION TITLE
Why is followers and the accounts that you follow public?

### DISCUSSION THREAD

**Discussion by Mr-Bajs** [mastodon/mastodon#23283] [General]
Why is the followers you have public for everyone to see? What's the reason behind this? Don't think that's nessecary for either for the function of mastodon and not in line with mastodon being more privacy oriented?

I don't think that your followers should be displayed at all for anyone else then yourself. And if you wanna display who you follow should be an Opt-in choice that is disabled by default.

**Comment by trwnh** [comment]
You do at least have an option to hide your network, but you are correct, this is opt-out and not opt-in. Personally I agree with you and also think it should be opt-in.

I think it mostly comes down to Mastodon being inspired by Twitter, and Twitter shows followers and follows, so Mastodon did/does too.

**Comment by ghost** [comment]
I couldn't agree more!

I wish I could get an option on the official Mastodon app to get rid of / hide statistics on my profile 💡I'd like people to follow me because they think we could have a good connection & interesting interactions, based on my profile and past toots, but not because *other* people are actually following me 😊 For example, I'm not subscribing to RSS feed because of any statistics about how many people are actually reading it 😉 We're not bots, we actually are people 🌟

And BTW, the fork glitch-soc has already implemented it: https://glitch-soc.github.io/docs/features/hide-follower-count/ so... this should not be an impossible option for the official Mastodon app to add it as well 😉

**Comment by Mr-Bajs** [comment]
The one and only valid reason is discoverablity i suppose. I wonder though if that's wanted or not. I guess somehow that's more or less a feature that mass media and advertisers is intrested in. Having some sort of metric of whos successful on platforms, but hence Mastodon instances isn's really ad-powerd then i don't see the reason. It should be opt in and not opt out.
